File: PE16/PE16.cpp
#include "pch.h"
#include <iostream>
#include "BigInteger.h"
using namespace std;

int main()
{
	int mysum = 0;
	string mystring = (BigInteger("2")).pow(1000).toString();
	for (char c : mystring) {
		mysum += c - '0';
	}
	cout << mysum << endl;
}

File: PE16/PE16.java
import java.math.*;

public class PE16 {
	public static void main(String args[]) {
		int mysum = 0;
		String mystring = (new BigInteger("2")).pow(1000).toString();
		for (char c : mystring.toCharArray()) {
			mysum += c - '0';
		}
		System.out.println(mysum);
	}
}
